apps/tailwind-components/components/input/Label.vue

```diff
@@ -1,28 +1,22 @@
 <script setup lang="ts">
-withDefaults(
-  defineProps<{
-    disabled?: boolean;
-    valid?: boolean;
-    hasError?: boolean;
-    required?: boolean;
-    hideLabel?: boolean;
-  }>(),
-  {
-    disabled: false,
-    hasError: false,
-    valid: false,
-    hideLabel: false,
-  }
-);
+defineProps<{
+  invalid?: boolean;
+  valid?: boolean;
+  disabled?: boolean;
+  required?: boolean;
+  hideLabel?: boolean;
+  for: string;
+}>();
 </script>
 
 <template>
   <label
     class="pl-3 text-body-base"
+    :for="for"
```

Comment:
syntax error here

Reply:
are you sure? I think it is the parser not accepting our prop is called 'for' too